core/server/lib/members/index.js

```diff
@@ -11,6 +11,20 @@ module.exports = function MembersApi() {
         return res.end(token);
     });
 
+    apiRouter.post('/signin', (req, res) => {
+        res.writeHead(200, {
+            'Set-Cookie': `signedin=true;HttpOnly;Max-Age=180;Path=/ghost/api/v2/members/token`
```

Comment:
This path needs to be dynamic - take into account blog subdomain - and also work independently - will solve once we start getting some configs i suppose